README.md

```diff
@@ -30,6 +30,29 @@ A maintained and comprehensive list of awesome Fuel resources!
 - [Sway Programming - An Introduction by 0xready](https://vertical.im/0xReady/sway-programming-an-introduction-0cG5tFSv)
 - [The Case for Modular Maxis](https://davidphelps.substack.com/p/the-case-for-modular-maxis)
 
+### International
+
+#### Russia
+- [Кошелек Burner Wallet + Fuel = 🔥](https://medium.com/@mririkoz_41994/%D0%BA%D0%BE%D1%88%D0%B5%D0%BB%D0%B5%D0%BA-burner-wallet-fuel-9c80cf6ad204)
+- [О проекте Fuel.](https://medium.com/@balamosha/%D0%B2%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5-e7dca0630332)
+- [Представляем “Beta-1”, первый модульный уровень исполнения, ориентированный на разработчиков Fuel Testnet](https://medium.com/@ixelaa/%D0%BF%D1%80%D0%B5%D0%B4%D1%81%D1%82%D0%B0%D0%B2%D0%BB%D1%8F%D0%B5%D0%BC-beta-1-%D0%BF%D0%B5%D1%80%D0%B2%D1%8B%D0%B9-%D0%BC%D0%BE%D0%B4%D1%83%D0%BB%D1%8C%D0%BD%D1%8B%D0%B9-%D1%83%D1%80%D0%BE%D0%B2%D0%B5%D0%BD%D1%8C-%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D0%BD%D0%B5%D0%BD%D0%B8%D1%8F-%D0%BE%D1%80%D0%B8%D0%B5%D0%BD%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D0%B9-%D0%BD%D0%B0-%D1%80%D0%B0%D0%B7%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%BE%D0%B2-fuel-ecf81da5d298)
+- [Масштабируемость Ethereum: почему все должны думать о Fuel](https://medium.com/@ixelaa/%D0%BC%D0%B0%D1%81%D1%88%D1%82%D0%B0%D0%B1%D0%B8%D1%80%D1%83%D0%B5%D0%BC%D0%BE%D1%81%D1%82%D1%8C-ethereum-%D0%BF%D0%BE%D1%87%D0%B5%D0%BC%D1%83-%D0%B2%D1%81%D0%B5-%D0%B4%D0%BE%D0%BB%D0%B6%D0%BD%D1%8B-%D0%B4%D1%83%D0%BC%D0%B0%D1%82%D1%8C-%D0%BE-fuel-f1d035020bd9)
+- [Testnet SwaySwap.](https://medium.com/@cryptocreepz/testnet-swayswap-f9c242e125eb)
+
+#### Chinese
```

Comment:
```suggestion
#### China
```
The PR is using the country name not the language as the topics, shouldn´t this one be `China`? Or change the other ones to the language?

Reply:
Yes, it's a mistake. It's been revised, sir.